core/src/test/java/com/datastax/oss/driver/internal/core/pool/ChannelPoolTestBase.java

```diff
@@ -111,17 +111,4 @@ DriverChannel newMockDriverChannel(int id) {
     when(driverChannel.toString()).thenReturn("channel" + id);
     return driverChannel;
   }
-
-  // Wait for all the tasks on the pool's admin executor to complete.
-  void waitForPendingAdminTasks() {
```

Comment:
why we don't need this method anymore?

Reply:
It doesn't work anymore with Netty 4.1.45.
So I replaced it with all the timed verifies / awaitility stuff.